src/containers/boards/reducer.js

```diff
@@ -1,41 +1,91 @@
-import { CLEAN_STATE, GET_ALL_BOARDS_FOR_USER, ADD_BOARD, DELETE_BOARD } from './constants'
+import { CLEAN_STATE, GET_OWNED_BOARDS_FOR_USER, GET_CONTRIBUTING_BOARDS_FOR_USER, ADD_BOARD, DELETE_BOARD } from './constants'
 
 const initialState = {
-  data: [],
-  areFetching: false,
-  areFetched: false,
+  ownedBoards: {
+    data: [],
+    areFetching: false,
+    areFetched: false,
+  },
+  contributingBoards: {
+    data: [],
+    areFetching: false,
+    areFetched: false,
+  },
 }
 
 export default (state = initialState, action) => {
   switch (action.type) {
     case CLEAN_STATE:
       return initialState
-    case `${GET_ALL_BOARDS_FOR_USER}`:
+    case `${GET_OWNED_BOARDS_FOR_USER}`:
```

Comment:
shouldn't areFetched be set to false ?

Reply:
Instead of saying "those boards have been successfully fetched", I've wanted to say "the request has been sent and there is a result". I will update the term.